src/LinuxCaptureService/LinuxCaptureService.cpp

```diff
@@ -17,16 +17,18 @@ grpc::Status LinuxCaptureService::Capture(
         reader_writer) {
   orbit_base::SetCurrentThreadName("CSImpl::Capture");
 
-  std::unique_ptr<orbit_capture_service_base::ClientCaptureEventCollectorBuilder>
-      client_capture_event_collector_builder =
-          orbit_capture_service_base::CreateGrpcClientCaptureEventCollectorBuilder(reader_writer);
+  std::unique_ptr<orbit_capture_service_base::ClientCaptureEventCollectorManager>
+      client_capture_event_collector_manager =
+          orbit_capture_service_base::CreateGrpcClientCaptureEventCollectorManager(reader_writer);
+  orbit_producer_event_processor::ClientCaptureEventCollector* client_capture_event_collector =
+      client_capture_event_collector_manager->GetClientCaptureEventCollector();
```

Comment:
What about instantiating the `GrpcClientCaptureEventCollector` directly here? I think the manager is not needed anymore. You could just write

```cpp
orbit_capture_service_base::GrpcClientCaptureEventCollector client_capture_event_collector{reader_writer};
```